controls/V-71961.rb

```diff
@@ -1,10 +1,10 @@
 # encoding: utf-8
 #
 
-grub_superusers = attribute(
-  'grub_superusers',
-  description: 'superusers for grub boot ( array )',
-  default: ['root']
+grub_superuser = attribute(
+  'grub_superuser',
+  description: 'superuser for grub boot',
+  default: 'root'
```

Comment:
Please ensure the tests will work with one or more users defined as a superuser.